pkgs/racket-doc/syntax/scribblings/parse/define.scrbl

```diff
@@ -38,6 +38,8 @@ Defines a macro named @racket[macro-id]; equivalent to the following:
 (fn2 a #:b 'c)
 ]
 
+@history[#:changed "6.12.0.3" "Change generated pattern head to (~var macro-name id) from macro-name"]
```

Comment:
I’d just tweak this note a little bit. You can use Scribble formatting here, so you can use things like `@racket`, and even if you don’t change anything else, `macro-name` should be `macro-id`. Still, I’d probably change this to something like:

```racket
@history[#:changed "6.12.0.3" @elem{Changed pattern head to @racket[(~var macro-id id)] from
                                    @racket[macro-id], allowing tilde-prefixed identifiers or
                                    identifiers containing colons to be used as @racket[macro-id]
                                    without producing a syntax error.}]
```

Reply:
Works for me. pushed.

Reply:
err....pushed that version.r